### PR TITLE
Verify Headless Services

### PR DESCRIPTION
**Pull request by waveywaves**
Added ability to verify headless service using the verifyService() function

Fixes https://github.com/openshift/jenkins-client-plugin/issues/217

cc @gabemontero

### CODE REVIEW

File: src/main/resources/com/openshift/jenkins/plugins/OpenShiftDSL.groovy [hunk 1]
```diff
@@ -945,28 +947,47 @@ class OpenShiftDSL implements Serializable {
...
-            } catch (IOException e) {
-                logToTaskListener("IOException " + e.getMessage() + " connecting to service " + svcName);
-                LOGGER.log(Level.FINE, "verifyService", e);
-                try {
-                    Thread.sleep(2500);
-                } catch (InterruptedException e1) {
+            if (ip == "None"){ 
+                InetAddress inetAddress = null;
+                try{
+                    inetAddress = InetAddress.getByName(svcName);
```
Comment (gabemontero) [inline src/main/resources/com/openshift/jenkins/plugins/OpenShiftDSL.groovy]: @waveywaves I've got a question in to our networking team on whether the dns lookup should work even if the jenkins pod is in a separate namespace than the headless service

Comment (gabemontero) [reply src/main/resources/com/openshift/jenkins/plugins/OpenShiftDSL.groovy]: I think it should be want to make sure.

In the interim, if you could construct such a test case and confirm/deny that fact, do that and respond here.

Comment (waveywaves) [reply src/main/resources/com/openshift/jenkins/plugins/OpenShiftDSL.groovy]: I will check if it works with services in other namespaces as well once and update it here along with the README.

**Pull request closed without merging**